video/out/vulkan/context_mac.m

```diff
@@ -56,6 +56,11 @@ static bool mac_vk_init(struct ra_ctx *ctx)
     struct mpvk_ctx *vk = &p->vk;
     int msgl = ctx->opts.probing ? MSGL_V : MSGL_ERR;
 
+    if (!NSApp) {
+        MP_ERR(ctx, "Failed to initialise macvk context, no NSApplication initialized.\n");
```

Comment:
I think we mostly us US English spelling. s/initialise/initialize/

Reply:
damn at least it's right in the commit message.